docs/API.md

```diff
@@ -36,6 +36,7 @@ All of the following options are optional.
 - `modes`, Object: over ride the default modes with your own. `MapboxDraw.modes` can be used to see the default values. More information on custom modes [can be found here](https://github.com/mapbox/mapbox-gl-draw/blob/master/docs/MODES.md).
 - `defaultMode`, String (default: `'simple_select'`): the mode (from `modes`) that user will first land in.
 - `userProperties`, boolean (default: `false`): properties of a feature will also be available for styling and prefixed with `user_`, e.g., `['==', 'user_custom_label', 'Example']`
+- `insertBeforeLayerId`, string (default: `undefined`): The ID of an existing layer to insert MapboxDraw draw layers before. If this argument is not specified, layers will be appended to the end of the layers array - [more here](https://docs.mapbox.com/mapbox-gl-js/api/#map#addlayer). 
```

Comment:
For parity with GL JS methods such as [`Map#addLayer`](https://docs.mapbox.com/mapbox-gl-js/api/map/#map#addlayer) and [`Map#moveLayer`](https://docs.mapbox.com/mapbox-gl-js/api/map/#map#movelayer), it might be clearer + conciser to name the option `beforeId` rather than `insertBeforeLayerId`.